src/Foundation/Features/MyAccount/ProfilePage/Index.cshtml

```diff
@@ -229,6 +229,6 @@
 </div>
 <div class="row">
     <div class="col-12">
-        @Html.PropertyFor(x => x.CurrentContent.ActivityFeed)
+        @*@Html.PropertyFor(x => x.CurrentContent.ActivityFeed)*@
```

Comment:
Why is this commented out?  Should it be removed?

Reply:
No, there are some bugs in Prorfile page and I missed them. I have updated.